cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -74,7 +74,7 @@
       cronjob at:'*/10 * * * *', do:deploy_dir('bin', 'cron', 'delete_twilio_data')
       cronjob at:'* * * * *', do:deploy_dir('bin', 'cron', 'confirm_usage')
       cronjob at:'5 16 * * *', do:deploy_dir('bin', 'cron', 'calculate_workshop_survey_results')
-      cronjob at:'20 6 * * *', do:deploy_dir('bin', 'cron', 'dms_tasks')
+      cronjob at:'20 6 * * 3,6', do:deploy_dir('bin', 'cron', 'dms_tasks')
```

Comment:
If I understand UTC correctly, this will start running around 11:20 PM pacific time. Could we switch to earlier (say, switch the hour to 1:20 UTC, or 6:20 PM pacific time) to have it closer to being done by the morning?